### PR TITLE
add instructions for atomic-run command on Fedora Atomic

### PR DESCRIPTION
**Pull request by andreasn**

### CODE REVIEW

**Comment by petervo** [comment]
The container is really only for connecting to that host directly with your browser. You can still add it to another cockpit without doing that. Is it worth explaining that a little more?

**Comment by andreasn** [comment]
Yeah, that would be good to add.
Something like:
"Connect to a Fedora Atomic host from an existing host running Cockpit with the Add Server UI.
Alternatively you can install Cockpit on the Atomic host with the command <code>atomic run cockpit/ws</code>"

File: running.html [hunk 1]
```diff
@@ -82,7 +82,8 @@
           </div>
           <div class="col-md-12 os-info os-info-fedoraatomic" style="display: none;">
             <div class="os-infobox">
-            Cockpit comes installed by default in Fedora Atomic.
+            Connect to a Fedora Atomic host from an existing host running Cockpit with the Add Server UI.<br/>
```
Comment (stefwalter) [inline running.html]: Suggest using the word 'using' instead of 'with' here. Makes it clearer.

File: running.html [hunk 2]
```diff
@@ -82,7 +82,8 @@
...
+            Alternatively you can install Cockpit on the Atomic host with the command <code>atomic run cockpit/ws</code>
```
Comment (stefwalter) [inline running.html]: You need a ```sudo``` in the beginning of that command.

**Comment by cgwalters** [comment]
See also https://fedorahosted.org/cloud/ticket/105

**Pull request merged**